docs/index.rst

```diff
@@ -0,0 +1,38 @@
+.. meta::
+    :description: HSA runtime implementation
+    :keywords: ROCR, ROCm, library, tool, runtime
+
+.. _index:
+
+=====================
+ROCR documentation
+=====================
+
+The ROCm runtime (ROCR) is AMD's implementation of HSA runtime, which is a thin, user-mode API that exposes the necessary interfaces to access and interact with graphics hardware driven by the AMDGPU driver set and the ROCK kernel driver. To learn more, see :ref:`what-is-rocr-runtime`
+
+You can access ROCR code on our `GitHub repository <https://github.com/ROCm/ROCR-Runtime>`_.
+
+The documentation is structured as follows:
+
+.. grid:: 2
+  :gutter: 3
+
+  .. grid-item-card:: Install
+
+    * :ref:`installation`
+    
+  .. grid-item-card:: API reference
+
+    * :ref:`c-interface-adaptors`
+    * :ref:`environment-variables`
+    * :ref:`api`
+
+  .. grid-item-card:: Contribution
+
+    * :ref:`contributing-to-rocr`
+
+To contribute to ROCR, refer to :ref:`contributing-to-rocr`.     
```

Comment:
Isn't this a duplicate of what is on top of the boxes?

Reply:
Yes, last time I, you and Randy had a discussion about confusion regarding the contributing to docs or the component and you had suggested adding this line.